HW11/helicopter.py

```diff
@@ -0,0 +1,43 @@
+from HW11.vehicle import Vehicle
+
+
+# Define the Helicopter class, which inherits from the Vehicle class
+# Inheritance
+
+class Helicopter(Vehicle):
+
+    def recharge(self):
+        pass
+
+    def __init__(self, max_speed, rotor_diameter, year, model):
+        super().__init__(max_speed)
+        self.rotor_diameter = rotor_diameter
+        self.year = year  # public attribute
+        self.__model = model  # private attribute
+
+    def __get_model(self):  # Private method
+        return self.__model
+
+    def get_model(self):  # public method which use private method
+        return self.__get_model()
```

Comment:
what is the purpose of this 2 methods? it just duplicate each other

Reply:
It's mistake, forgot to delete one of them